component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/reporter/GenericInputValidator.java

```diff
@@ -86,6 +87,8 @@ public class GenericInputValidator {
                     new AbstractMap.SimpleImmutableEntry<>(APPLICATION_ID, String.class),
                     new AbstractMap.SimpleImmutableEntry<>(APPLICATION_NAME, String.class),
                     new AbstractMap.SimpleImmutableEntry<>(APPLICATION_OWNER, String.class),
+                    new AbstractMap.SimpleImmutableEntry<>(ORGANIZATION_ID, String.class),
```

Comment:
Please make sure there new attributes are set by MI to the builder. Else publishing will fail for even the Choreo flow